sonaranalyzer-dotnet/its/expected/akka.net/Akka-{5DEDDF90-37F0-48D3-A0B0-A5CBD8A7E377}-S4226.json

```diff
@@ -108,357 +43,6 @@
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  107,
-"startColumn":  35,
-"endLine":  107,
-"endColumn":  67
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  122,
-"startColumn":  35,
-"endLine":  122,
-"endColumn":  67
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  138,
-"startColumn":  35,
-"endLine":  138,
-"endColumn":  57
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  152,
-"startColumn":  35,
-"endLine":  152,
-"endColumn":  57
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  169,
-"startColumn":  35,
-"endLine":  169,
-"endColumn":  63
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Scheduler\SchedulerExtensions.cs",
-"region":  {
-"startLine":  184,
-"startColumn":  35,
-"endLine":  184,
-"endColumn":  63
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Stash\StashFactory.cs",
-"region":  {
-"startLine":  19,
-"startColumn":  30,
-"endLine":  19,
-"endColumn":  41
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Stash\StashFactory.cs",
-"region":  {
-"startLine":  25,
-"startColumn":  30,
-"endLine":  25,
-"endColumn":  41
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
-"uri":  "sources\akka.net\src\core\Akka\Actor\Stash\StashFactory.cs",
-"region":  {
-"startLine":  30,
-"startColumn":  30,
-"endLine":  30,
-"endColumn":  41
-}
-}
-},
-{
-"id":  "S4226",
-"message":  "Either move this extension to another namespace or move the method inside the type itself.",
-"location":  {
 "uri":  "sources\akka.net\src\core\Akka\Actor\SupervisorStrategy.cs",
```

Comment:
this is an enum and should not be reported, I don't understand why it is reported

Reply:
I forgot to run update-expected :(